### ISSUE TITLE
Exception message in English

### ISSUE THREAD

**Issue by devaskim** [xingshuangs/iot-communication#16]
Hi @xingshuangs and thanks for the awesome tool!

1. Do you have any plans to migrate to English description of any exception message?
2. Or do you consider contribution of such changes in the project?

**Comment by xingshuangs** [comment]
Good idea, and I hope everyone can give me better advice. But the comments section is a bit too much work.

**Comment by devaskim** [comment]
@xingshuangs Thanks for quick response
Just to clarify by description or message of an exception I meant this
```
throw new S7CommException("S7参数中只有[type]=字节和字符串类型数据的[count]才能大于1，其他必须等于1");
```

Thanks in advance

**Comment by xingshuangs** [comment]
Completed in v1.5.1

**Comment by devaskim** [comment]
Hi @xingshuangs,
Could you also translate in English error codes in 
https://github.com/xingshuangs/iot-communication/blob/master/src/main/java/com/github/xingshuangs/iot/protocol/s7/enums/EReturnCode.java ?

P.S.: I thought Github issues have `Reopen` button.

**Comment by xingshuangs** [comment]
Completed in v1.5.2
@devaskim
